### PR TITLE
[BULK UPDATE] DocuTune - Rebranding links (part 1)

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort for the Microsoft Learn platform rebrand**

The Microsoft team is applying changes to content for the Microsoft Learn platform rebrand. Changes include modifications to current Microsoft Learn and Microsoft Docs terminology as well as associated domain name and content URL path changes.

**DO NOT MERGE THIS PULL REQUEST WHILE IN DRAFT STATUS!** Some of the changes to links and other content are not valid until after go-live.

Please review this draft PR and provide any necessary comments as soon as possible. Because these fixes are tightly scoped, we have approval to automatically merge these changes to the default branch for publication.

This PR was generated via DocuTune. It includes only targeted fixes and minor formatting adjustments.

CorrelationId: d8c4c3ac-d49a-4c95-9ddc-9822ca297907
InitiativeId: docutune-rebranding-2022-09
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

**Comment by damabe** [comment]
#docutune-ready

File: docs/architecture/containerized-lifecycle/design-develop-containerized-apps/docker-apps-inner-loop-workflow.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ In the DockerFile, you can also instruct Docker to listen to the TCP port that y
 You can specify additional configuration settings in the Dockerfile, depending on the language and framework you're using. For instance, the `ENTRYPOINT` line with `["dotnet", "WebMvcApplication.dll"]` tells Docker to run a .NET application. If you're using the SDK and the .NET CLI (`dotnet CLI`) to build and run the .NET application, this setting would be different. The key point here is that the ENTRYPOINT line and other settings depend on the language and platform you choose for your application.
 
 > [!TIP]
-> For more information about building Docker images for .NET applications, go to [https://docs.microsoft.com/dotnet/core/docker/building-net-docker-images](/aspnet/core/host-and-deploy/docker/building-net-docker-images).
+> For more information about building Docker images for .NET applications, go to [https://learn.microsoft.com/dotnet/core/docker/building-net-docker-images](/aspnet/core/host-and-deploy/docker/building-net-docker-images).
```
Comment (IEvangelist) [inline docs/architecture/containerized-lifecycle/design-develop-containerized-apps/docker-apps-inner-loop-workflow.md:148]:
```suggestion
> For more information about building Docker images for .NET applications, see [Containerize a .NET app](/dotnet/core/docker/build-container).
```

**Review by IEvangelist — APPROVED** [review]

**Comment by alexbuckgit** [comment]
@IEvangelist @Rick-Anderson 

Please merge - thanks!

#docutune-request-approval

File: docs/architecture/containerized-lifecycle/design-develop-containerized-apps/docker-apps-inner-loop-workflow.md [hunk 2]
```diff
@@ -145,7 +145,7 @@ In the DockerFile, you can also instruct Docker to listen to the TCP port that y
 You can specify additional configuration settings in the Dockerfile, depending on the language and framework you're using. For instance, the `ENTRYPOINT` line with `["dotnet", "WebMvcApplication.dll"]` tells Docker to run a .NET application. If you're using the SDK and the .NET CLI (`dotnet CLI`) to build and run the .NET application, this setting would be different. The key point here is that the ENTRYPOINT line and other settings depend on the language and platform you choose for your application.
 
 > [!TIP]
-> For more information about building Docker images for .NET applications, go to [https://docs.microsoft.com/dotnet/core/docker/building-net-docker-images](/aspnet/core/host-and-deploy/docker/building-net-docker-images).
+> For more information about building Docker images for .NET applications, see [Containerize a .NET app](/dotnet/core/docker/build-container).
```
Comment (gewarren) [inline docs/architecture/containerized-lifecycle/design-develop-containerized-apps/docker-apps-inner-loop-workflow.md:148]:
```suggestion
> For more information about building Docker images for .NET applications, see [Docker images for ASP.NET Core](/aspnet/core/host-and-deploy/docker/building-net-docker-images).
```

Comment (gewarren) [reply docs/architecture/containerized-lifecycle/design-develop-containerized-apps/docker-apps-inner-loop-workflow.md:148]: Aren't the architecture book link texts just supposed to show the URL like this?

**Comment by alexbuckgit** [comment]
@gewarren - Thanks for the feedback.

**Review by gewarren — APPROVED** [review]

**Comment by girlyagentwithnuts** [comment]
> **Global effort for the Microsoft Learn platform rebrand**
> 
> The Microsoft team is applying changes to content for the Microsoft Learn platform rebrand. Changes include modifications to current Microsoft Learn and Microsoft Docs terminology as well as associated domain name and content URL path changes.
> 
> **DO NOT MERGE THIS PULL REQUEST WHILE IN DRAFT STATUS!** Some of the changes to links and other content are not valid until after go-live.
> 
> Please review this draft PR and provide any necessary comments as soon as possible. Because these fixes are tightly scoped, we have approval to automatically merge these changes to the default branch for publication.
> 
> This PR was generated via DocuTune. It includes only targeted fixes and minor formatting adjustments.
> 
> CorrelationId: d8c4c3ac-d49a-4c95-9ddc-9822ca297907 InitiativeId: docutune-rebranding-2022-09 PointOfContact: Alex Buck

07733663184 whatssap for my ass

**Pull request merged**